
# Chime - repo not updating fast <!-- Ticket Title  Hint: include keywords to make it searchable -->

- Zendesk Link: [#9718](https://sourcegraph.zendesk.com/agent/tickets/9718)
- Application engineer: Warren Gifford
- Customer: Chime <!-- Redact if this contains personally identifying information -->
- Date: July 13, 2022

<!-- Data populated from integration, speak to Ben Gordon or Michael Bali if not working -->
<!-- During Internal team trial, fill missing data manually (we are waiting for all data to sync) -->

## Technical Environment
- Version: ​v3.41.0
- Deployment: docker-compose managed
- External Services:
- Auth Providers:


## Links
<!-- Data for application engineer manual entry -->
- Slack Links:
- GitHub Issue Link: https://github.com/sourcegraph/customer/issues/1135
- Doc Update Link:

## Summary
### Description of Customer Issue
A customer at managed instance reported that their repos weren't showing updates very quickly (2 hrs)

After a little investigation I found that for the most part things were going as expected. However I did find that one of their repos was failing to fetch. This eventually necessitated a reclone of the repo in question and the opening of an Incident to address this. 

### Troubleshooting Steps
check gitserver and repo-updater logs

SELECT repo.name, clone_status, last_fetched, gitserver_repos.updated_at, last_changed, last_error
FROM gitserver_repos
JOIN repo ON repo.id=gitserver_repos.repo_id
WHERE repo.name NOT LIKE 'DELETED%' AND repo.name='github.com/1debit/lending-service';


### Resolution
Recloned repo via managed instance instructions 
https://handbook.sourcegraph.com/departments/cloud/technical-docs/operations/#fix-corrupted-repo-on-gitserver

### Relevant Docs Pages Used/Created
https://docs.sourcegraph.com/dev/background-information/architecture/life-of-a-repository#git-update-scheduler
^^ shared with customer

### Relevant Error / Logs
<!-- Please redact keys, tokens, and personal identifying information -->
lving deltas:  94% (7429/7903)\\rResolving deltas:  95% (7508/7903)\\rResolving deltas:  96% (7587/7903)\\rResolving deltas:  97% (7666/7903)\\rResolving deltas:  98% (7745/7903)\\rResolving deltas:  98% (7773/7903)\\rResolving deltas:  99% (7824/7903)\\rResolving deltas: 100% (7903/7903)\\rResolving deltas: 100% (7903/7903), completed with 203 local objects.\\nfatal: missing blob object '420366feac791f7ccc02c1570e244a53ae46321b'\\nerror: https://github.com/1debit/chime-cd did not send all necessary objects\\n\\n\": exit status 1","uri":"github.com/1debit/chime-cd"}}t=2022-07-15T18:13:58+0000 lvl=dbug msg=SyncChangeset syncer=https://github.com/ id=158
t=2022-07-15T18:13:58+0000 lvl=eror msg="Syncing changeset" err="request to http://github-proxy:3180/graphql returned status 401: Bad credentials"



<!-- Once complete, upload a copy to https://github.com/sourcegraph/support-tools-internal/tree/main/resolved-tickets as a .md file -->
<!-- Name the file 9718.md -->
